content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/transformation.yaml

```diff
@@ -10,25 +10,9 @@ object:
   fields:
   - state_machine_phase:
       value: transformation
-  - cleanup_state_machine:
-      value: "/Transformation/StateMachines/VMTransformation/TransformationCleanup"
   - State5:
       value: "/Transformation/Common/WaitForHandover"
-      on_entry: /System/CommonMethods/MiqAe.WeightedUpdateStatus(weight => 1, description
-        => "Waiting for Handover", task_message => "Pre-migration")
-      on_exit: /System/CommonMethods/MiqAe.WeightedUpdateStatus(weight => 1, description
-        => "Waiting for Handover", task_message => "Pre-migration")
-      on_error: /System/CommonMethods/MiqAe.WeightedUpdateStatus(weight => 1, description
-        => "Waiting for Handover", task_message => "Pre-migration")
+      on_entry: "/System/CommonMethods/MiqAe.WeightedUpdateStatus"
+      on_exit: "/System/CommonMethods/MiqAe.WeightedUpdateStatus"
+      on_error: "/System/CommonMethods/MiqAe.WeightedUpdateStatus"
```

Comment:
@fdupont-redhat Why are you retaining these calls?  What does the method do when passed without parameters?

Reply:
The method should do nothing but allow cancellation, which would be useless at this step. I removed the calls.

Reply:
@fdupont-redhat Which calls did you remove?  These are still "mostly" here.  🙂

Reply:
Hum. Weird. When I look at 'Files changed' tab, I can see they are gone.
Grep'ing `WeightedUpdateStatus` in the repo only return the definition of the method under /System/CommonMethods/MiqAe.

Reply:
I'm seeing all deletes now.  Not sure if I had a stale page open before.  👍